packages/extension/src/background/handlers/Tabs.ts

```diff
@@ -72,6 +73,28 @@ export default class Tabs {
     return this.#state.sign(url, new RequestExtrinsicSign(request), { address, ...pair.meta });
   }
 
+  private rpcSend (request: RequestRpcSend): Promise<JsonRpcResponse> {
+    return this.#state.rpcSend(request);
```

Comment:
if the page sends a request and the user closes the page before the response arrives, the background script is going to throw (line 34)

https://github.com/polkadot-js/extension/blob/602dfdc0262b62e142bfa47163b93b262a184d49/packages/extension/src/background/handlers/index.ts#L26-L40

Reply:
proposing #254